### PR TITLE
Fix capitalization inconsistency in optim.rst

### PR DESCRIPTION
**Pull request by ssnl**

### CODE REVIEW

File: docs/source/optim.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ Algorithms
 .. autoclass:: SGD
     :members:
 
-How to adjust Learning Rate
+How to adjust learning rate
```
Comment (xuhdev) [inline docs/source/optim.rst]: Why? Seems other section titles also do not use capital letters except the first letter

Comment (SsnL) [reply docs/source/optim.rst]: Which is exactly why I did this...

Comment (xuhdev) [reply docs/source/optim.rst]: Right... I saw it backwards XD

**Review by xuhdev — APPROVED** [review]

**Pull request closed without merging**